internal/services/springcloud/spring_cloud_service_resource.go

```diff
@@ -531,7 +543,9 @@ func resourceSpringCloudServiceRead(d *pluginsdk.ResourceData, meta interface{})
 
 	configServer, err := configServersClient.Get(ctx, id.ResourceGroup, id.SpringName)
 	if err != nil {
-		return fmt.Errorf("retrieving config server settings for %s: %+v", id, err)
+		if !utils.ResponseWasBadRequest(configServer.Response) {
```

Comment:
and again why are we ignoring a bad request? we should wrap this in a check for the sku?